BackofficeBundle/ContentAttributeEvents.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle;
+/**
+ * Class ContentAttributesEvents
+ */
+class ContentAttributeEvents
```

Comment:
As every other document event definition, you should put this class in the model-interface library